src/helpers/renderableChildren.js

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { isFragment } from 'react-is';
 
 export default function renderableChildren(childrenProp) {
-  return React.Children.toArray(childrenProp).filter((child) => child === 0 || child);
+  return React.Children.toArray(childrenProp).reduce((children, child) => {
+    if (isFragment(child)) {
+      return [...children, ...renderableChildren(child.props.children)];
+    }
+
+    if (child === 0 || child) {
+      return [...children, child];
```

Comment:
```suggestion
      return children.concat(renderableChildren(child.props.children));
    }

    if (child === 0 || child) {
      return children.concat(child);
```
no need to use the much slower spread when concat exists :-)

Reply:
Resolved by 8331555